### DISCUSSION TITLE
Feature Request: Show the discussions from the special repository on profile page

### DISCUSSION THREAD

**Discussion by SVilgelm** [community/community#69575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

it would be great to show the discussions tab from the special repository (if enabled) on the profile page. 
Like `projects` or `packages` tabs:
<img width="793" alt="Screenshot 2023-10-08 at 10 12 50 PM" src="https://github.com/community/community/assets/523825/11c30f1b-5041-4fd8-87b9-252fbff49532">

thank you!

**Comment by Pasan-Pahasara** [comment]
Response #2

**Reply by SVilgelm** [reply]
How the feature request of better project support is related to this?
I'm asking to display the discussion on the profile page, it is totally unrelated to the projects
